docs/preprocessing.md

```diff
@@ -30,7 +30,7 @@ This dataset adheres to the following requirements:
 
 - Data should be stored in netcdf files, one file per day.
 - Importantly, the midnight of the next day should also be included (except
-  perhaps for the last day)
+  perhaps for the last day) #TODO: @Peter, this is not correct anymore, is it?
```

Comment:
You're right, I can rewrite that. I'll push some commits to this branch if that's okay with you